examples/bn254/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "bn254-inner"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+bn254-core = { path = "../../core" }
+risc0-zkvm = { path = "../../../../risc0/zkvm", default-features = false, features = ["std", "unstable"] }
+substrate-bn = "0.6.0"
+
+[patch.crates-io]
+# TODO: Get a tag stabilized and use it
```

Comment:
was this a TODO you meant to leave?

Reply:
Yes. Until we have made a tagged release for https://github.com/risc0/paritytech-bn/tree/release/v0.6.0, this TODO needs to remain, and we can't make that tagged release until risc0-bigint2 v1.3 has been released